pywhy_stats/categorical.py

```diff
@@ -0,0 +1,293 @@
+"""Independence test among categorical variables using power-divergence tests.
+
+Works on categorical random variables. Based on the ``lambda_`` parameter, one
+can compute a wide variety of different categorical hypothesis tests.
+
+Examples
+--------
+>>> import pywhy_stats as ps
+>>> res = ps.categorical.ind([1, 2, 3], [4, 5, 6])
+>>> print(res.pvalue)
+>>> 1.0
+"""
+
+import logging
+from typing import Optional
+
+import numpy as np
+from numpy.typing import ArrayLike
+from scipy import stats
+from sklearn.preprocessing import LabelEncoder
+
+from .pvalue_result import PValueResult
+
+
+def ind(X: ArrayLike, Y: ArrayLike, lambda_: str = "cressie-read") -> PValueResult:
+    """Perform an independence test using power divergence test.
```

Comment:
The overview of the `lambda_` is great, but a bit confusing what this means (without reading the paper). What about renaming it `method` or having a better description how this prefix `lambda_` relates to these other values without needing to read the paper.

Reply:
Changed to `method`.